Dockerfile

```diff
@@ -26,15 +26,15 @@ FROM ubuntu:20.04 AS release
 LABEL maintainer="hans.hjort@xandr.com" 
 RUN apt-get update && \
     apt-get install --assume-yes apt-utils && \
```

Comment:
Do we need `--no-install-recommends` here too?

Reply:
I removed the entire line. Tested locally, I can build and run the image and run local http request.